### DISCUSSION TITLE
How to get ArgoCD apply kustomization regardlessly?

### DISCUSSION THREAD

**Discussion by Magicloud** [argoproj/argo-cd#13532] [Q&A]
I have a kustomization containing only one `configMapGenerator`, from some data files. The generated configmaps are used in other kustomization, so I have to disable the random suffix.

At first time, ArgoCD loaded it and applied it. But then when I updated the data files. ArgoCD saw the commits, and told me the project was synced. But the configmap did not get updated (both content and age).

Manually applying the kustomization got it updated. So the kustomization is correct, but ArgoCD did not see the change in data files.

**Comment by emirot** [comment]
I have similar questions regarding the suffix hash generated from SecretGenerator and Configmap generator.

Is it still needed when using ArgoCD ?

What I could get is that it would automatically trigger a rolling update of a deployment whenever the contents of the configmap/secret changes. 

So I guess ArgoCD just update the resources but does not do a rolling update?
